include/deal.II/dofs/dof_accessor.templates.h

```diff
@@ -1980,10 +1980,6 @@ template <int spacedim, bool level_dof_access>
 inline unsigned int
 DoFAccessor<0, 1, spacedim, level_dof_access>::n_active_fe_indices() const
 {
-  Assert((std::is_same<DoFHandler<1, spacedim>,
-                       dealii::DoFHandler<1, spacedim>>::value == true),
-         ExcNotImplemented());
```

Comment:
That is an interesting remnant of the `hp::DoFHandler` days :)

Reply:
It is a pretty funny assertion!